embulk-jruby-strptime/src/main/java/org/embulk/spi/time/StrptimeFormat.java

```diff
@@ -0,0 +1,44 @@
+package org.embulk.spi.time;
+
+// Ported from org.jruby.util.RubyDateFormatter.Format in JRuby
```

Comment:
If it's a kind of copy, is licensing okay? We may want/need some notations on licensing.